app/jobs/absolute_id_create_record_job.rb

```diff
@@ -0,0 +1,179 @@
+# frozen_string_literal: true
+class AbsoluteIdCreateRecordJob < ApplicationJob
```

Comment:
This class is untested: https://1347-165745983-gh.circle-artifacts.com/0/coverage/index.html#d46fab0128696ee76db1970302d5be0a0905ae06